EU4toV2/Source/V2World/Flags/Flags.cpp

```diff
@@ -61,32 +58,34 @@ void V2::V2Flags::SetV2Tags(const std::map<std::string, V2Country*>& V2Countries
 					randomCK2title = countryMapper.getCK2TitleMapper().getRandomIndianFlag();
 				}
 
-				if (randomCK2title && (usableFlagTags.find(*randomCK2title) != usableFlagTags.end()))
+				if (randomCK2title && usableFlagTags.count(*randomCK2title))
 				{
-					tagMap[i->first] =* randomCK2title;
+					tagMap[country.first] = *randomCK2title;
 					usableFlagTags.erase(*randomCK2title);
-					requiredTags.erase(i->first);
+					requiredTags.erase(country.first);
 				}
 			}
 		}
 	}
 
-	for (std::set<std::string>::iterator i = usableFlagTags.begin(); i != usableFlagTags.end();)
+	// Drop all incoming flag titles with _ in second spot (k_bavaria)
+	for (auto flagItr = usableFlagTags.begin(); flagItr != usableFlagTags.end();)

```

Comment:
done

Reply:
correction, remove_if cannot be used on sets and this is a set because it needs unique tags. Google says the current method is actually still current.

Reply:
Ah, pity.